rerun_cpp/src/rerun/archetypes/transform3d.hpp

```diff
@@ -13,6 +13,42 @@
 namespace rerun {
     namespace archetypes {
         /// A 3D transform.
+        ///
+        /// ## Example
+        ///
+        ///```ignore
+        ///// Log some transforms.
+        ///
+        /// #include <rerun.hpp>
+        ///
+        /// #include <cmath>
+        ///
+        /// namespace rr = rerun;
+        /// namespace rrd = rr::datatypes;
+        ///
+        /// const float pi = static_cast<float>(M_PI);
+        ///
+        /// int main() {
+        ///    auto rr_stream = rr::RecordingStream(\"transform3d\");
+        ///    rr_stream.connect(\"127.0.0.1:9876\");
+        ///
+        ///    auto arrow = rr::archetypes::Arrows3D({0.0f, 1.0f, 0.0f});
+        ///
+        ///    rr_stream.log(\"base\", arrow);
+        ///
+        ///    rr_stream.log(\"base/translated\", rr::archetypes::Transform3D({1.0f, 0.0f, 0.0f}));
+        ///    rr_stream.log(\"base/translated\", arrow);
+        ///
+        ///    rr_stream.log(
+        ///        \"base/rotated_scaled\",
+        ///        rr::archetypes::Transform3D(
+        ///            rrd::RotationAxisAngle({0.0f, 0.0f, 1.0f}, rrd::Angle::radians(pi / 4.0f)),
+        ///            2.0f
+        ///        )
+        ///    );
+        ///    rr_stream.log(\"base/rotated_scaled\", arrow);
+        /// }
```

Comment:
Those backslashes are a serious issue; the whole point of docstring examples is to be easily copy-pastable... :disappointed:

Reply:
Hmm yes. I'll fix that in an upcoming PR